universal-application-tool-0.0.1/test/support/ResourceCreator.java

```diff
@@ -17,6 +17,7 @@
 import services.question.types.QuestionDefinition;
 import services.question.types.TextQuestionDefinition;
 
+/** Creates services that interface with the default database server for testing. */
```

Comment:
(blocking) let's keep this PR scope tight. You're welcome to submit a different PR to add comments and fix formatting.